spice/deps/helper_cb.py

```diff
@@ -31,7 +31,7 @@
 
 
 logging.basicConfig(level=logging.DEBUG)
-logger = logging.getLogger(__name__)
+logger = logging.getLogger(__file__)
 
```

Comment:
log should be more clear now:
__name__ = always `main` 
__file__ variable contains filename of the script